pkg/term/term_windows.go

```diff
@@ -1,12 +1,15 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	// "github.com/Sirupsen/logrus"
```

Comment:
All references are commented out. If you do not comment it out, Go complains about an unused package. I can remove the line...and all commented-out calls to logrus, but it seemed better to leave them in.